extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/VertxOAuth2IdentityProvider.java

```diff
@@ -8,6 +8,7 @@
 import org.jose4j.jwt.JwtClaims;
 import org.jose4j.jwt.consumer.InvalidJwtException;
 
+import io.quarkus.oidc.VertxJwtCallerPrincipal;
```

Comment:
I'm not convinced by the name of this class. It's exposed to the user AFAICS, could we find a better name? I don't see why we have `Vertx` in it typically. It looks historical to me. `OidcJwtCallerPrincipal` maybe?

I will do the changes. Just asking for advice.